src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtilsUnitTest.java

```diff
@@ -105,40 +297,103 @@
 
     }
 
+    @DataProvider
+    private Object[][] provideForGetDataSourceVersionString() {
+        return new Object[][] {
+                {
+                        DataSourceUtils.MIN_MAJOR_VERSION_NUMBER,
+                        DataSourceUtils.MIN_MINOR_VERSION_NUMBER,
+                        DataSourceUtils.MIN_DATE,
+                        "v1.6.20190124"
+                },
+                {
+                        DataSourceUtils.MAX_MAJOR_VERSION_NUMBER,
+                        DataSourceUtils.MAX_MINOR_VERSION_NUMBER,
+                        DataSourceUtils.MAX_DATE,
+                        "v1.7.20200521"
```

Comment:
These will also break when the versions are updated -- recommend writing everything in terms of the defined constants.

Reply:
Actually these won't break when the versions are updated - this is just checking the string formatting of the version numbers, so it should be fine regardless as to what versions we create in the future.

Reply:
No, it will definitely break as soon as MAX_MAJOR_VERSION_NUMBER is changed to something other than 1, etc., since you've hardcoded the string `v1.7.20200521`

Reply:
Ah - right.  I misunderstood.  Fixed.